code/form/style.css

```diff
@@ -0,0 +1,116 @@
+/*** Header ***/
+
+header {
+  background-color: white;
+  height: 80px;
+  display: flex;
+  position: sticky;
+  top: 0;
+  z-index: 100;
+  align-items: center;
+  justify-content: space-between;
+  overflow: hidden;
+  padding: 0 16px;
+  opacity: 0.98;
+}
+
+header img {
+  width: 180px;
+}
+
+img[src="./img/user_icon_flaticon.png"] {
+  width: 35px;
+}
+
+/*** Body ***/
+
+body {

```

Comment:
Traditionally, the body-tag is placed at the top, as it's the wrapper of all visible contents. `<header>`-tag is not to be confused with `<head>`, that contains title, some scripts and other "non-visible" things. :)